TheJumpCloud/support
# Sa 2380 wifi profile example

## Issues
* [SA-2380](https://jumpcloud.atlassian.net/browse/SA-2380) - Windows Wifi Profile Via Commands Example

## What does this solve?

Adds an example workflow for admins wishing to deploy a plaintext passcode wifi profile to systems in their org via commands.

## Is there anything particularly tricky?

A physical windows system or virtualized WIFI enabled VM is needed to test this profile

## How should this be tested?

Export a wifi profile via the instructions in the blog post, create a command and deploy to a system without that profile. The system should be able to select that wifi profile without entering the password.

## Screenshots
